packages/flutter/lib/src/material/tabs.dart

```diff
@@ -734,6 +735,9 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
   /// share of the available space.
   final bool isScrollable;
 
+  /// The amount of space by which to inset the tab bar.
+  final EdgeInsetsGeometry? padding;
```

Comment:
Let's document how this behaves when scrollable is true or not. I understand now that when the TabBar is _not_ scrollable, it is no different than just wrapping it in Padding. But when TabBar.scrollable is true then this is necessary to achieve the desired effect of the Padding _being part of the scrollable_. This was a bit nuanced and hard to understand, so we should make it clearer in the docs. :)

Reply:
Do you have any recommendations? I can change it the way you want it 👍